onnxruntime/core/providers/qnn/qnn_provider_factory.cc

```diff
@@ -17,6 +17,25 @@ struct QNNProviderFactory : IExecutionProviderFactory {
     return std::make_unique<QNNExecutionProvider>(provider_options_map_, config_options_);
   }
 
+  std::unique_ptr<IExecutionProvider> CreateProvider(const OrtSessionOptions* session_options,
+                                                     const OrtLogger* logger) override {
+    ORT_UNUSED_PARAMETER(logger);
+    const ConfigOptions& config_options = session_options->GetConfigOptions();
+    const std::unordered_map<std::string, std::string>& config_options_map = config_options.GetConfigOptionsMap();
+
+    // The implementation of the SessionOptionsAppendExecutionProvider C API function automatically adds EP options to
+    // the session option configurations with the key prefix "EP_NAME:".
```

Comment:
Can we change to 'ep.QNN.' or does that require more discussion? Doesn't need to be in this PR but the sooner the better.

Reply:
I think we just need to consult with @ivberg. The SessionOptionsAppendExecutionProvider implementation is using this specific prefix (`"EP_NAME:"`) for ETW logging added by this PR: https://github.com/microsoft/onnxruntime/pull/19397

Want to make sure it's ok for us to change

Reply:
Thanks for checking. Yea there is nothing dependent on the name format right now, so feel free to change the format.

I do like proposed new format